modules/images/regenerate-existing-images/helper.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Functions used by the background process API.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Creates a background job.
+ *
+ * Job refers to the term and queue refers to the `background_job` taxonomy.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $action Name of job action.
+ * @param array  $data Optional. Arbitrary data for the job. Default empty array.
```

Comment:
We follow style for param doc. https://github.com/WordPress/performance/blob/trunk/modules/images/webp-uploads/helper.php#L61-L65

```suggestion
 * @param string $action Name of job action.
 * @param array  $data   Optional. Arbitrary data for the job. Default empty array.
```